upgrading/os_upgrades.adoc

```diff
@@ -35,7 +35,7 @@ that all of the rules in other services are correct.
 +
 However, instead of rebooting a node host, you can restart the services that are
 affected, or preserve the `iptables` state. Both processes are described in the
-xref:../../admin_guide/iptables.adoc#admin-guide-iptables[{product-title}
+xref:../admin_guide/iptables.adoc#admin-guide-iptables[{product-title}
 IPtables] topic. The `ovs` flow rules do not need to be saved, but restarting
```

Comment:
total nit, but this should be iptables instead of IPtables to be consistent with the style of the page it is pointing to